checkmate/views/status.py

```diff
@@ -1,10 +1,10 @@
-"""The status end-point required to allow us to work with AWS."""
-
 from pyramid.view import view_config
+from sentry_sdk import capture_message
 
 
-@view_config(route_name="get_status", renderer="json", http_cache=0)
-def get_status(_request):
-    """Status endpoint."""
+@view_config(route_name="status", renderer="json", http_cache=0)
+def status(request):
```

Comment:
Renamed the view to just `status` as well. This view config actually lacks `request_method` so it can be called with any HTTP status. But also, `status()` is a simpler name and is consistent with how this view is named in our other apps. You don't really need views named things like `get_*()` and `post_*()` unless you're actually going to multiple views for the same route, and when that happens you can easily rename the views at that time. Until then it's simpler for the path, route and view to all just have the same simple name